EndlessClient/Network/UnknownEntitiesRequester.cs

```diff
@@ -0,0 +1,104 @@
+﻿using EndlessClient.GameExecution;
+using EOLib.Domain.Map;
+using EOLib.Net;
+using EOLib.Net.Communication;
+using Microsoft.Xna.Framework;
+using System;
+
+namespace EndlessClient.Network
+{
+    public class UnknownEntitiesRequester : GameComponent
+    {
+        private readonly ICurrentMapStateRepository _currentMapStateRepository;
+        private readonly IPacketSendService _packetSendService;
+        private DateTime _lastRequestTime;
+        private const double REQUEST_INTERVAL_SECONDS = 1;
```

Comment:
How did you decide the interval for this? Is this what the original client does?

Reply:
It was an arbitrary guess. I'm not really sure exactly how the official client does it. It seems instant but it makes sense to queue them up rather than send a request for every single unknown.

Can play around with the interval on a busy map and see how the traffic looks once reoserv has npcs lol